extension/persistence/eclipselink/src/main/java/io/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -243,7 +243,7 @@ public <T> T runInTransaction(
         return result;
       } catch (Exception e) {
         tr.rollback();
-        LOG.debug("transaction rolled back: {}", e);
+        LOG.debug("transaction rolled back", e);
```

Comment:
Thanks for spotting this @ebyhr !

Still, I'm not sure the fix follows the original intention. I suppose the idea was to log `Exception.toString()` as part of the fist line of the log message... WDYT about `LOG.debug("transaction rolled back: {}", e, e);`

Reply:
That's right. It was try to use `LOG.debug(String format, Object arg)`. 

But I think this change is also fine since we will have the details in the stack trace.

Can you also make similar change in line 281  to` LOG.debug("transaction rolled back", e);` ?

Reply:
fair enough (but sometimes it can be helpful to see the message on the same line when you grep for `transaction rolled back` :slightly_smiling_face: )